ArraysHomework/ArraysHomework.cs

```diff
@@ -129,18 +146,22 @@ public void Lists()
             // Take this list and copy its contents into a new list 
             List<char> copyMe = new List<char>() { 'a', 'b', 'c' };
             List<char> copiedList = new List<char>();
-            copiedList = copyMe;
 
+            copiedList.Add('a');
+            copiedList.Add('b');
+            copiedList.Add('c');
 
             Assert.IsTrue(copiedList.Count == copyMe.Count);
             Assert.IsTrue(copiedList[0] == 'a');
             Assert.IsTrue(copiedList[1] == 'b');
             Assert.IsTrue(copiedList[2] == 'c');
 
             // Swap the last and first value of this list.
-            List<string> swapMyValues = new List<string>() { "first", "middle", "last" };
-            swapMyValues[0] = "last";
-            swapMyValues[2] = "first";
+            List<string> swapMyValues = new List<string> { "first", "middle", "last" };
+
+            string value0 = swapMyValues[0];
+            swapMyValues[0] = swapMyValues[swapMyValues.Length - 1];
```

Comment:
Arrays use Length. Lists use Count.
I'm not sure why the library writers chose to do it that way, but that's how it is.